packages/scheduler/src/__tests__/SchedulerUMDBundle-test.internal.js

```diff
@@ -17,7 +17,8 @@ describe('Scheduling UMD bundle', () => {
   });
 
   function filterPrivateKeys(name) {
-    return !name.startsWith('_');
+    // TODO: Figure out how to forward priority levels.
```

Comment:
Probably should inline them? Also should use Symbols, with a fallback to magic numbers.